test/client/components/script/script.test.js

```diff
@@ -0,0 +1,78 @@
+import { fetchScript, fetchScriptQuestions } from '../../../../public/src/actions/script';
+import fixtures from '../../client_fixtures';
+import { checkObjectProps } from '../../client_test_helpers.js';
+
+const { current_script,
+        scriptQuestionsFixture: questions } = fixtures.scriptViewFixtures;
+
+const props = {
+  id: 1,
+  current_script,
+  questions,
+  fetchScript,
+  fetchScriptQuestions
+};
+
+describe('<ScriptPage />', () => {
```

Comment:
Are you going to add in a shallow render and then test the props or skip that for now?
@allieglotfelty